apps/condo/domains/ticket/schema/TicketReportService.js

```diff
@@ -95,6 +127,93 @@ const TicketReportService = new GQLCustomSchema('TicketReportService', {
                 return { data }
             },
         },
+        {
+            access: access.canReadTicketReportAnalyticsData,
+            schema: 'getTicketReportAnalyticsData(data: TicketReportAnalyticsInput!): TicketReportAnalyticsOutput',
+            resolver: async (parent, args, context, info, extra) => {
+                const { dateFrom, dateTo, groupBy, userOrganizationId, ticketType, viewMode, addressList } = args.data
+                const isLineChart = viewMode === 'line'
+                const statuses = await getOrganizationStatuses(context, userOrganizationId)
+                const statusesMap = Object.fromEntries(statuses.map(({ type, name }) => ([type, name])))
+                const userProperties = await getOrganizationProperties(context, userOrganizationId)
+                const userPropertiesMap = Object.fromEntries(userProperties.map(({ id, address }) => ([id, address])))
+                const daysCount = moment(dateTo).diff(moment(dateFrom), 'days') + 1
```

Comment:
why we add '+1'?

Reply:
This solution is shortest way to fill array with dates in between period with including current date